applications/openshift/authentication/idp_is_configured/rule.yml

```diff
@@ -59,7 +59,7 @@ references:
   cis@ocp4: 3.1.1
   nerc-cip: CIP-004-6 R2.2.2,CIP-004-6 R2.2.3,CIP-007-3 R.1.3,CIP-007-3 R5,CIP-007-3 R5.1,CIP-007-3 R5.1.1,CIP-007-3 R5.1.2,CIP-007-3 R5.1.3,CIP-007-3 R5.2,CIP-007-3 R5.2.1,CIP-007-3 R5.2.3,CIP-007-3 R5.3.1,CIP-007-3 R5.3.2,CIP-007-3 R5.3.3
   nist: AC-2,AC-2(1),AC-2(2),AC-2(3),AC-2(4),AC-2(5),AC-2(6),AC-2(7),AC-2(8),AC-7
-  pcidss: Req-2.2
+  pcidss: Req-2.2, Req-8.1.1
```

Comment:
This makes the build fail with:
```
 ValueError: Comma-separated 'pcidss' reference in /content/products/ocp4/../../applications/openshift/authentication/idp_is_configured/rule.yml contains whitespace. 
 ```
Let's remove the whitespace.

Reply:
Ok will do